### ISSUE TITLE
ImportError: No module named packages.urllib3.poolmanager

### ISSUE THREAD

**Issue by RGaonkar** [sigmavirus24/requests-toolbelt#82]
I am using the Flickrapi which makes use of the requests-toolbelt utilities. The following import error occurred while using it. Any pointers on what should be done? Here is the trace back

Traceback (most recent call last):
  >File "search4_new.py", line 6, in <module>
  >  import flickrapi
  >File "build/bdist.linux-x86_64/egg/flickrapi/\__init\__.py", line 52, in <module>
  >File "build/bdist.linux-x86_64/egg/flickrapi/core.py", line 13, in <module>
  >File "build/bdist.linux-x86_64/egg/flickrapi/tokencache.py", line 10, in <module>
  >File "build/bdist.linux-x86_64/egg/flickrapi/auth.py", line 23, in <module>
  >File "build/bdist.linux-x86_64/egg/requests_toolbelt/\__init\__.py", line 12, in <module>
  >File "build/bdist.linux-x86_64/egg/requests_toolbelt/adapters/\__init\__.py", line 12, in <module>
  >File "build/bdist.linux-x86_64/egg/requests_toolbelt/adapters/ssl.py", line 13, in <module>
  >  SSLError -- exception raised for I/O errors
>ImportError: No module named packages.urllib3.poolmanager

**Comment by sigmavirus24** [comment]
Can you provide any information about the version of requests and the requests_toolbelt installed?

**Comment by sigmavirus24** [comment]
Also, can you provide details about how you installed requests and the requests-toolbelt

**Comment by pbkhrv** [comment]
I'm having the same problem. Running this on Ubuntu 14.04.1. Installed requests-toolbelt via pip.

Here's what I got from pip:

```
> pip list
argparse (1.2.1)
boto (2.38.0)
chardet (2.0.1)
colorama (0.2.5)
Flask (0.10.1)
html5lib (0.999)
itsdangerous (0.24)
Jinja2 (2.7.3)
MarkupSafe (0.23)
Pillow (2.6.1)
pip (1.5.4)
requests (2.2.1)
requests-toolbelt (0.4.0)
setuptools (3.3)
six (1.5.2)
urllib3 (1.7.1)
Werkzeug (0.10.4)
wheel (0.24.0)
wsgiref (0.1.2)
```

```
> pip show requests
Name: requests
Version: 2.2.1
Location: /usr/lib/python2.7/dist-packages
Requires:
```

```
> pip show requests-toolbelt
Name: requests-toolbelt
Version: 0.4.0
Location: /usr/local/lib/python2.7/dist-packages
Requires: requests
```

**Comment by pbkhrv** [comment]
Upgrading "requests" to 2.4.3 doesn't help. On Ubuntu 14.x, if you do "pip install requests==2.4.3", it breaks pip completely (something about unable to import ReadIncomplete)

For me, the fix was to switch to Ubuntu 15.04.

**Comment by sigmavirus24** [comment]
Oh the problem is apt-installed packages. I'll just give in and fix this here too. *sigh*
